torch/csrc/autograd/utils/wrap_outputs.h

```diff
@@ -81,6 +81,12 @@ inline PyObject* wrap(double value) {
   return PyFloat_FromDouble(value);
 }
 
+inline PyObject* wrap(std::complex<double> value) {
+  // I could probably also use FromComplex with a reinterpret cast,
```

Comment:
lol what input type does FromComplex expect?

Reply:
It can also take a Py_complex which is defined to be:

```
typedef struct {
   double real;
   double imag;
} Py_complex;
```